forge/tests/error/main.rkt

```diff
@@ -80,8 +80,9 @@
     (list "failed_theorem.frg" #rx"failed.")
     (list "failed_sat.frg" #rx"Failed test")
     (list "failed_unsat.frg" #rx"Failed test")
-    (list "properties_undirected_tree_underconstraint_error.frg" #rx"isUndirectedTree implies TreeWithEdges failed. Found instance")
-    (list "properties_undirected_tree_overconstraint_error.frg" #rx"isUndirected implies isUndirectedTree failed. Found instance")
+    (list "failed_sat.frg" #rx"Failed test") 
+    (list "properties_undirected_tree_underconstraint_error.frg" #rx"(isUndirectedTree implies TreeWithEdges) failed. Found instance")
```

Comment:
`#rx` is a regular expression, not a plain string. But what about a totally different error message? It's an `assert` now. `Assertion 'A is necessary for B' failed. Found instance` ?

cc @tnelson

Reply:
I like this. I'm not sure how to intercept and change the error message though, since Forge thinks this is a theorem.

Reply:
Oh, right. Then let's stick to `Theorem` for now.

Reply:
You can always check the error tests locally by running `racket main.rkt`. To speed it up, comment out the other tests.